spec/lib/services/sku-pack-service-spec.js

```diff
@@ -151,7 +152,7 @@ describe("SKU Pack Service", function() {
         });
     });
 
-    describe('hould post a sku', function(){
+    describe('should post a sku', function(){
```

Comment:
How about to add a new test case to cover the new code? verify that the installPack will be rejected if the sku has already existed?